<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CodeImportOptionsPage</class>
 <widget class="QWidget" name="CodeImportOptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>244</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="title">
        <string>Include search paths</string>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>513</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Additional include search paths could be added by specifying environment variable UMBRELLO_INCPATH.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>C++ Import</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="createArtifactCheckBox">
            <property name="text">
             <string>For each imported file create an artifact in the component view</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="resolveDependenciesCheckBox">
            <property name="text">
             <string>Resolve dependencies (reduces import speed)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="supportCPP11CheckBox">
            <property name="text">
             <string>Enable C++11 standard (experimental)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
